scripts/update-ukci-kernels.py

```
@@ -0,0 +1,432 @@
+            hash_value,
+        ],
+        check=True,
+        text=True,
+        stdout=subprocess.PIPE,
+    )
+    return result.stdout.strip()
+
+
+def prefetch_rc_hash(source_url: str, version: str) -> str:
+    filename = f"linux-{version}.tar.gz"
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)